src/core/skills/skill.ts

```diff
@@ -97,7 +97,7 @@ export abstract class Skill {
   public getAnimationSteps(
     event: ServerEventFinder<GameEventIdentifiers.SkillUseEvent | GameEventIdentifiers.CardUseEvent>,
   ): EventProcessSteps {
-    if (EventPacker.getIdentifier(event) === GameEventIdentifiers.CardUseEvent) {
+    if (EventPacker.getIdentifier(event) === GameEventIdentifiers.SkillUseEvent) {
```

Comment:
I guess this must be 
```suggestion
    if (EventPacker.getIdentifier(event) === GameEventIdentifiers.SkillUseEvent || EventPacker.getIdentifier(event) === GameEventIdentifiers.CardUseEvent) {
```